src/components/OrderHistory/OrderHistoryListStyles.ts

```diff
@@ -11,6 +11,6 @@ export const StyledOrderHistoryList = styled.div`
 `;
 
 export const BtnsContainer = styled(Container)`
-  margin: auto 2rem;
-  justify-content: end;
+  margin: 1rem 2rem 2rem 2rem;
+  justify-content: start;
```

Comment:
I'm not sure "start" does anything here. It should be at least flex-start if it's a flex container. And it's the default value anyway